crates/biome_grit_patterns/tests/specs/ts/identifier.snap

```diff
@@ -0,0 +1,47 @@
+---
+source: crates/biome_grit_patterns/tests/spec_tests.rs
+expression: identifier
+---
+SnapshotResult {
+    messages: [],
+    matched_ranges: [
+        Range {
+            start: Position {
+                line: 1,
+                column: 4,
+            },
+            end: Position {
+                line: 1,
+                column: 4,
+            },
+            start_byte: 25,
+            end_byte: 28,
+        },
```

Comment:
The snapshot already highlights a bug, since the calculated line and column numbers are always the same, even if the byte ranges are correct. I didn't want to start bugfixing in the same PR however, so this will be a follow-up fix :)